chanzuckerberg/single-cell-data-portal
# feat(wmg): Clear All Genes Button

## Reason for Change

- #4849

## Changes

- add
    - Add Clear Genes button
- remove
- modify

## Testing steps

- Select gene(s) (tissue does not need to be selected)
- Clear Genes button should appear when at least one gene is selected
- Click Clear Genes button
- There should be no more selected genes

## Notes for Reviewer

https://clear-genes-frontend.rdev.single-cell.czi.technology/gene-expression
